packages/earth-admin/src/utils/hooks.tsx

```diff
@@ -145,3 +149,28 @@ export function mergePages(pagedResponse: any[]): IMergedResults {
     { data: [] }
   );
 }
+
+/**
+ * Queries the API and returns dynamic filters grouped by their keys
+ */
+export function useFilters(resource: string, fetcher: (any) => Promise<any>, options: object = {}) {
+  const { selectedGroup } = useAuth0();
+  const query = {
+    select: 'name,slug',
```

Comment:
y u need name & slug?

Reply:
I need a field that's present on every request. Definitely don't need both. I'll change it to 'id'